jemoji.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'jemoji'
   s.summary     = 'GitHub-flavored emoji plugin for Jekyll'
-  s.version     = '0.8.0'
+  s.version     = '0.8.1'
```

Comment:
We do this. Please revert this change.

Reply:
I don't have the branch anymore, feel free to commit the same change and delete this PR, it'll be OK